smart-contracts/osmosis/contracts/cl-vault/src/helpers/getters.rs

```diff
@@ -27,8 +27,7 @@ pub fn get_range_admin(deps: Deps) -> Result<Addr, ContractError> {
 pub fn get_asset0_value(
     storage: &dyn Storage,
     querier: &QuerierWrapper,
-    token0: Uint128,
-    token1: Uint128,
+    tokens_provided: (Uint128, Uint128),
```

Comment:
I would prefer to keep the arguments as before and rename the function to `get_token0_value`. Providing pairs is not very readable and in this case relies on the the implicit convention that asset0 refers to the first entry in tokens_provided.